stdlib/LibGit2_jll/Project.toml

```
@@ -1,6 +1,6 @@
 name = "LibGit2_jll"
 uuid = "e37daf67-58a4-590a-8e99-b0245dd2ffc5"
-version = "1.3.0+0"
+version = "1.4.3+0"
```

Changing this to
```suggestion
version = "1.4.3+1"
```
(ref: https://github.com/JuliaPackaging/Yggdrasil/pull/4931) and refreshing again the checksums should hopefully fix the test failure.